libs/cdk/zone-configurations/src/lib/zone-config.ts

```diff
@@ -148,11 +129,7 @@ function createZoneFlagsConfigurator(): ZoneConfig {
 }
 
 /**
- * An object for typed zone-flags configuration. It exposes flags under the following scopes:
- * - global (global environment API)
- * - events (event listener names and behavior)
- * - runtime (configurable behaviour at runtime)
- * - test (patches for testing libs like jasmine, jest etc)
+ * An object for typed zone-flags configuration.
```

Comment:
Why did you remove the comments here?

Reply:
I didn’t, I just copied the file from your PR.

Reply:
managed to update, just need to push to your PR later on